Automatic CP workflow is busted (`PULL_REQUEST_CP_TEMPLATE.md`)

[Example failure](https://github.com/flutter/flutter/actions/runs/17277066741/job/49036351656#step:7:111):

```txt
open ../PULL_REQUEST_CP_TEMPLATE.md: no such file or directory
Error: Process completed with exit code 1.
Error: Unable to process file command 'env' successfully.
Error: Invalid value. Matching delimiter not found 'EOF'
##[debug]System.Exception: Invalid value. Matching delimiter not found 'EOF'
##[debug]   at GitHub.Runner.Worker.EnvFileKeyValuePairs.GetEnumerator()+MoveNext()
##[debug]   at GitHub.Runner.Worker.SetEnvFileCommand.ProcessCommand(IExecutionContext context, String filePath, ContainerInfo container)
##[debug]   at GitHub.Runner.Worker.FileCommandManager.ProcessFiles(IExecutionContext context, ContainerInfo container)
##[debug]Finishing: Create PR on CP success
```

Cause was refactor of supposedly unnecessary code in https://github.com/flutter/flutter/pull/174448.

--- a/.github/workflows/easy-cp.yml
+++ b/.github/workflows/easy-cp.yml
@@ -72,7 +72,7 @@ jobs:
           git push https://${{ env.GITHUB_TOKEN }}@github.com/flutteractionsbot/flutter cp-${CHANNEL}-${COMMIT_SHA}
           {
             echo 'PR_URL<<EOF'
-            gh pr create --title "[CP-${CHANNEL}]${PR_TITLE}" --body-file ../PULL_REQUEST_CP_TEMPLATE.md --base ${RELEASE_BRANCH} --label "cp: review" --repo flutter/flutter --head flutteractionsbot:cp-${CHANNEL}-${COMMIT_SHA}
+            gh pr create --title "[CP-${CHANNEL}]${PR_TITLE}" --body-file ".github/PR_TEMPLATE/PULL_REQUEST_CP_TEMPLATE.md" --base ${RELEASE_BRANCH} --label "cp: review" --repo flutter/flutter --head flutteractionsbot:cp-${CHANNEL}-${COMMIT_SHA}
             echo EOF
           } >> "$GITHUB_ENV"
         env:

